README.md

```diff
@@ -1,37 +1,81 @@
-# vrs-python-testing
-## Goal
-Provide functionality to connect variant-level VCF information to clinical cancer variant evidence. Also specing out the capabilities of variant normalization with `vrs-python` and evidence gathering with `metakb`
+<img width="685" alt="image" src="https://github.com/ohsu-comp-bio/vrs-python-testing/assets/47808/909db052-972c-4508-a2f4-8a389de03320">
 
-## Setup
-### Terra
-```
-cd ~
-git clone https://github.com/ohsu-comp-bio/vrs-python-testing.git
-cd vrs-python-testing
-gsutil cp 1000g_patient_na12878_evidence.ipynb $WORKSPACE_BUCKET/notebooks/
-```
 
-Once you open the notebook, it'll be copied down to your persistent disk in `"$HOME/$WORKSPACE/edit`, so you'll need to copy it back to the repo before committing any changes, eg
-```
-cp `"$HOME/$WORKSPACE/edit/1000g_patient_na12878_evidence.ipynb` $HOME/vrs-python-testing/
-```
+# VRS AnVIL
 
-### Local
-```
-bash setup.sh
-```
+## Project Overview
 
-## Testing
-```
+This Python project is designed to process Variant Call Format (VCF) files or other sources of variant information and perform lookup operations on Genomic Variation Representation Service (GA4GH VRS) identifiers. The GA4GH VRS identifiers provide a standardized way to represent genomic variations, making it easier to exchange and share genomic information.
+
+In addition, this project facilitates the retrieval of evidence associated with genomic alleles by leveraging the Genomic Data Representation and Knowledge Base (GA4GH MetaKB) service. GA4GH MetaKB provides a comprehensive knowledge base that links genomic variants to relevant evidence, enabling users to access valuable information about genomic alleles.
+
+## Features
+
+1. **VCF File Processing:**
+   - The project includes modules for reading and parsing VCF files, extracting relevant genomic information.
```

Comment:
Not sure if you had a vision in mind for this, right now our VCF processing is limited and specific to 1000G and is really just calling bcftools. Let me know.

Reply:
Please suggest re-phrase, but don't get down to the level of specific dataset or tools

Reply:
did a small change, I think the modules was the only part that needed changing